doc/source/ray-air/examples/gptj_batch_prediction.ipynb

```diff
@@ -0,0 +1,224 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# GPT-J6-B Batch Prediction with Ray AIR\n",
+    "\n",
+    "In this example, we will showcase how to use the Ray AIR for **GPT-J batch inference**. GPT-J is a GPT-2-like causal language model trained on the Pile dataset. This particular model has 6 billion parameters.\n",
```

Comment:
Can you add some tip about, they should first read about Key Concepts from AIR and Data?

Reply:
like "Prerequisite Material"